devices/Chs6540/Chs6540.nuspec

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>nanoFramework.Iot.Device.Chs6540</id>
+    <version>$version$</version>
+    <title>nanoFramework.Iot.Device.Chs6540</title>
+    <authors>nanoFramework project contributors</authors>
+    <owners>nanoFramework,dotnetfoundation</owners>
```

Comment:
This property is deprecated. We're not using it anymore.
```suggestion
```

Reply:
shall we remove everywhere?

Reply:
Same as above.